util/envDynawo.sh

```diff
@@ -1454,86 +1484,113 @@ deploy_dynawo() {
   fi
   cd $DYNAWO_DEPLOY_DIR
 
-  mkdir -p 3rdParty/sundials/include
-  mkdir -p 3rdParty/adept/include
-  mkdir -p 3rdParty/suitesparse/include
-  mkdir -p 3rdParty/nicslu/include
-  mkdir -p extraLibs/BOOST/include
-  mkdir -p extraLibs/LIBARCHIVE/include
-  mkdir -p extraLibs/LIBZIP/include
-  mkdir -p extraLibs/LIBXML/include
-  mkdir -p extraLibs/LIBIIDM/include
-  cp -R -P $DYNAWO_SUNDIALS_INSTALL_DIR/include/* 3rdParty/sundials/include/
-  cp -R -P $DYNAWO_ADEPT_INSTALL_DIR/include/* 3rdParty/adept/include/
-  cp -P $DYNAWO_SUITESPARSE_INSTALL_DIR/include/*.* 3rdParty/suitesparse/include/
+  mkdir -p include
+  cp -R -P $DYNAWO_SUNDIALS_INSTALL_DIR/include/* include/
```

Comment:
This might create issues if there are file name conflicts.

Reply:
I don't seehow it can happen

Reply:
for example sundials and adept both define a common.h

Reply:
(it is not the case, it is just to illustrate)

Reply:
I don't see any common.h in my sources but in any case each library usually puts its include in a folder and not directly in the include, so we need to include "sundials/file.h" or "adept/file.h"

Reply:
Yes, but it might happen anytime as we do not control how 3rd party library are installed

Reply:
In this case we could also have trouble at compile time so I don't see why this is a problem... Linux packages are installed this way on the system and you might run in the same trouble if you install packages that have the same include...

Reply:
Added -n option